nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -88,13 +106,27 @@ def __init__(
         self.dropout = nn.Dropout(dropout)
         self.norm_out = LayerNorm(d_model)
 
-    def forward(self, x, att_mask=None, pos_emb=None, pad_mask=None):
+    def forward(
+        self,
+        x,
+        att_mask=None,
+        pos_emb=None,
```

Comment:
Add types to input function arguments

Reply:
This a forward method and all inputs are torch.tensor.